src/components/Input.vue

```diff
@@ -0,0 +1,20 @@
+<script setup lang="ts">
```

Comment:
We better avoid using one word component, specially being same as the native one.